_inc/lib/class-jetpack-podcast-helper.php

```diff
@@ -193,16 +201,17 @@ private static function setup_tracks_callback( SimplePie_Item $episode ) {
 			'link'        => esc_url( $episode->get_link() ),
 			'src'         => esc_url( $enclosure->link ),
 			'type'        => esc_attr( $enclosure->type ),
-			'description' => self::get_plain_text( $episode->get_description() ),
-			'title'       => self::get_plain_text( $episode->get_title() ),
+			'description' => $this->get_plain_text( $episode->get_description() ),
+			'title'       => $this->get_plain_text( $episode->get_title() ),
+			'guid'        => $this->get_plain_text( $episode->get_id() ),
```

Comment:
@mmtr I added a commit to add the GUID for D53816-code. I haven't tested this but per the SimplePie docs

```
	/**
	 * Get the unique identifier for the item
	 *
	 * This is usually used when writing code to check for new items in a feed.
	 *
	 * Uses `<atom:id>`, `<guid>`, `<dc:identifier>` or the `about` attribute
	 * for RDF. If none of these are supplied (or `$hash` is true), creates an
	 * MD5 hash based on the permalink, title and content.
	 *
```